### ISSUE TITLE
Improve compatibility with on-scree readers

### ISSUE THREAD

**Issue by josuah** [brilliantlabsAR/noa-flutter#114]
> the app does not work well with VoiceOver on the iPhone. 
> Some of the buttons do not speak. These issues could be easily fixed by 
> a developer familiar with Apple's accessibility guidelines

Apparently, some particular buttons could need a few fixes.
Using https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios might help spot it.
